src/main/java/auxiliary/ColorSupplier.java

```diff
@@ -0,0 +1,17 @@
+package auxiliary;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private String color;
```

Comment:
let's remove this field from ColorSupplier 
ColorSupplier is a service class, so it shouldn't have any fields